### PR TITLE
chore: update issue template with browser status section

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds <strike>an optional dropdown where the author can choose a browser if it's known that the requested feature is supported in one browser</strike> a textarea for authors to add implementation notes with links to bug trackers, if possible.

Also add links to criteria for inclusion and experimental status pages on MDN.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/content-or-feature-suggestion.yml [hunk 1]
```diff
@@ -6,22 +6,39 @@ body:
...
+  - type: dropdown
+    id: browser
+    attributes:
+      label: Browser support
+      description: Is this feature available in a stable browser release?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
```
Comment (Rumyra) [inline .github/ISSUE_TEMPLATE/content-or-feature-suggestion.yml:27]: For this one, can we have an open text answer and also ask _what_ version of the browser the feature was released in, and any supporting links (like the browser implementation bug)

**Review by Rumyra — CHANGES_REQUESTED** [review]

**Comment by Rumyra** [comment]
Thank you so much for doing this @bsmth :D Just one suggestion

File: .github/ISSUE_TEMPLATE/content-or-feature-suggestion.yml (hunk 1 above)
Comment (bsmth) [reply .github/ISSUE_TEMPLATE/content-or-feature-suggestion.yml:27]: Can do indeed

Comment (bsmth) [reply .github/ISSUE_TEMPLATE/content-or-feature-suggestion.yml]: Done in https://github.com/mdn/mdn/pull/477/commits/8600f5d5ed950b9e750fa5b3e4be0be93af4245e

**Review by Rumyra — APPROVED** [review]

**Pull request merged**